eclair-core/src/main/scala/fr/acinq/eclair/io/Switchboard.scala

```diff
@@ -76,6 +77,8 @@ class Switchboard(nodeParams: NodeParams, router: ActorRef, watcher: ActorRef, r
 
     case 'peers => sender ! context.children
 
+    case GetRouterPeerConf => sender ! RouterPeerConf(nodeParams.routerConf, nodeParams.peerConnectionConf)
```

Comment:
This seems to be unused, and what's the purpose exactly? Seems that any "entity" (i.e plugin) that can send akka messages to the switchboard probably have a `NodeParams` available and doesn't need that.